packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumber } from '@defichain/jellyfish-json'
+import { OP_CODES, Script, TransactionSegWit, CreateMasternode, ResignMasterNode } from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+
+export class TxnBuilderMasternode extends P2WPKHTxnBuilder {
+  /**
+   * Build create masternode transaction
+   *
+   * @param {CreateMasternode} createMasternode transaction to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @return {Promise<TransactionSegWit>}
+   */
+  async create (createMasternode: CreateMasternode, changeScript: Script): Promise<TransactionSegWit> {
+    return await this.createDeFiTx(
+      OP_CODES.OP_DEFI_TX_CREATE_MASTER_NODE(createMasternode),
+      changeScript,
+      new BigNumber('1') // for creation fee (regtest - 1, other than regtest - 10)
```

Comment:
Missing env handling? In test/prod, it might be insufficient balance in collect prevout and fail right?

Reply:
yea you got the point.. hmm.. thinking about it

Reply:
the easiest approach for now
https://github.com/DeFiCh/jellyfish/blob/92c3ebc4ae14424ea69a85759af9ae54c8847fc5/packages/jellyfish-transaction-builder/src/txn/txn_builder_masternode.ts#L13-L20